### PR TITLE
Add functionality to set keyboard height 

### PR DESCRIPTION
**Pull request by jacobkapitein**
This is my first open source contribution, so please give any feedback regarding code, set-up of the pull request and the process.

I missed functionality to set the height of the keyboard. I used other keyboard before and switched to this one, due to the simplicity. The keyboard however, was a lot smaller than I was used to so I created this functionality.

In the pull request I tried following all rules noted [here](https://github.com/SimpleMobileTools/General-Discussion#contribution-rules-for-developers).

There are now 3 modes to set the keyboard: Small, Medium and Big/Large. It's just a really simple setting with a list of all options available. I tried different methods, including a textbox, radio buttons and the one implemented now: a pop-up with some options, comparable to the language selection.

Again: I really wanted this functionality and decided to maybe help out and implement it myself. Hope you like it.

If any code needs to be edited, changed, refactored or anything, please let me know. I am open to feedback.

### CODE REVIEW

File: app/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -27,6 +27,10 @@
     <string name="keycode_shift">Shift</string>
     <string name="keycode_enter">Enter</string>
     <!-- Settings -->
+    <string name="keyboard_height_multiplier">Keyboard height</string>
+    <string name="settings_keyboard_height_multiplier_option_small">Small</string>
```
Comment (tibbi) [inline app/src/main/res/values-ar/strings.xml:31]: havent checked the code yet, but you have these 3 strings in Simple Commons already

**Comment by tibbi** [comment]
resolve the conflicts please, meanwhile I already added the keyboard_height string to main repository so people can translate it

**Comment by jacobkapitein** [comment]
I will resolve the conflict, fix the issues and update the pull request before this weekend.

I assume you checked the code already then?

**Comment by tibbi** [comment]
not yet, didnt get to that part directly

**Comment by jacobkapitein** [comment]
Resolved the conflicts and resolved your comments.

**Comment by tibbi** [comment]
looks good, will see if this is what people expected :) Thanks

**Pull request merged**